autograder/autograder/grade_item.py

```diff
@@ -9,6 +9,294 @@
 from . import autograding_utils, testcase
 from . execution_environments import jailed_sandbox
 
+def get_item_from_item_pool(complete_config_obj, item_name):
+    for item in complete_config_obj['item_pool']:
+        if item['item_name'] == item_name:
+            return item
+    return None
+
+
+def get_all_testcases(
+    complete_config_obj,
+    config,
+    queue_obj,
+    working_directory,
+    which_untrusted,
+    item_name
+):
+    testcase_objs = []
+    testcase_specs = complete_config_obj['testcases']
+
+    for item in complete_config_obj['item_pool']:
+        testcase_specs += item_obj['testcases']
```

Comment:
If there are multiple test cases in the item, does it add an array of multiple test cases or does it append them as individual test cases?  (the latter I hope/assume)

Reply:
Yes, this returns the union of the two lists.